benchmark/README.md

```diff
@@ -1 +1,3 @@
-Benchmark is located in a separate repository: https://github.com/ClickHouse/ClickBench
+**Note:** ClickBench is located in a separate repository: https://github.com/ClickHouse/ClickBench
+
+This directory was created for micro-benchmarks (supposedly based on [google benchmark](https://github.com/google/benchmark)).
```

Comment:
If we will put microbenchmarks near the code - no need to change this directory.